impl-endpoints/cloudflare-go/runner.go

```diff
@@ -85,55 +88,101 @@ func (t testCaseDC) ServerConfig() (*tls.Config, error) {
 	return config, nil
 }
 
-func (t testCaseDC) ClientConnectionHandler(conn *tls.Conn) error {
+func (t testCaseDC) ClientConnectionHandler(conn *tls.Conn, err error) error {
+	if err != nil {
+		return fmt.Errorf("Unexpected error: \"%s\"")
+	}
 	if conn.ConnectionState().VerifiedDC == nil {
 		return errors.New("Failed to verify a delegated credential")
 	}
 	return nil
 }
 
-func (t testCaseDC) ServerConnectionHandler(conn *tls.Conn) error {
-	// TODO
-	return errors.New("NOT IMPLEMENTED")
+func (t testCaseDC) ServerConnectionHandler(conn *tls.Conn, err error) error {
+	panic("test case not implemented")
 }
 
-type echTestResult struct {
-	serverStatus tls.EXP_EventECHServerStatus
-}
+// The "ech-accept" test case in which a client offers the ECH extension and
+// the server accepts. The client verifies server name "backend.com". See
+// draft-ietf-tls-esni.
+//
+// TODO(cjpatton): Check via validatepcap that:
```

Comment:
Either before or after this PR lands, let's track this TODO with an issue.

Reply:
Done: https://github.com/xvzcf/tls-interop-runner/issues/38.